pages/documents/[slug].tsx

```diff
@@ -46,22 +47,20 @@ const UniteLegaleForAgentPage: NextPageWithLayout<IProps> = ({
           <HorizontalSeparator />
         </>
       )}
-      {isAgent(session) && <ActesSection uniteLegale={uniteLegale} />}
+      <DocumentWall
+        isVisible={isAgent(session)}
+        title="Actes et statuts"
+        id="actes"
+        uniteLegale={uniteLegale}
+      >
+        <DocumentActesSection uniteLegale={uniteLegale} />
+      </DocumentWall>
```

Comment:
@johangirod would like your input here. not a big fan of current implementation. How would you do that ?

Reply:
I think it's okay. The other way would be to add this logic to the `Section` component to factorize title, administration and such. I think we can postpone the decision until next time we need that logic.